tests/e2e/performance/csi_tests/test_pvc_bulk_clone_performance.py

```diff
@@ -159,6 +241,27 @@ def test_bulk_clone_performance(self, namespace, tmp_path):
                 f"for {self.interface} clone in bulk of {pvc_count} clones."
             )
 
+            # Produce ES report
+            # Collecting environment information
+            self.get_env_info()
+
+            # Initialize the results doc file.
+            full_results = self.init_full_results(
+                ResultsAnalyse(self.uuid, self.crd_data, self.full_log_path)
+            )
+
+            full_results.add_key("interface", self.interface)
+            full_results.add_key("bulk_size", pvc_count)
+            full_results.add_key("clone_size", vol_size)
+            full_results.add_key("bulk_creation_time", total_time)
+            full_results.add_key("data_size(MB)", total_files_size)
+            full_results.add_key("speed", speed)
+
+            # Write the test results into the ES server
+            full_results.es_write()
```

Comment:
I wonder if we could add the details of the elasticsearch path as well, so anyone interested could have a look at the ES data with one click.

Reply:
this can be done by : 
```
        if full_results.es_write():
            log.info(f"The Result can be found at : {full_results.results_link()}")
```

Reply:
@keesturam of course it is possible to add the results path link. 

Would you like it to be a part of the report in the Perf Dashbpoard?

Reply:
@keesturam added     

        full_results.add_key("es_results_link", full_results.results_link())

Reply:
perfect